config/tips.txt

```diff
@@ -132,3 +132,73 @@ Changelings can spawn as jobs that might otherwise not be spawn-able as a Traito
 Spacesuits have injection ports to administer medicine safely while in low pressure environments. These however take extra time to use.
 Setting your suit sensors to maximum isn't a good idea if you are an antagonist, as medbay can track wherever you go.
+Wrapping a bedsheet around a glass shard makes it easier to hold.
+Painkillers do not actually heal you - they just keep you standing.
+If you don't have a hazard vest, you can make armor using rail assemblies to hold together steel sheets - be aware of electrical conduction.
+A rag can hold some liquid - smothering someone's mouth will force them to breathe it in...
+You can take apart scissors with a screwdriver.
+Roll two newspapers together to swat away pests - then put a shard or a scissor blade in it...
+Syringes are delicate - do not harmfully stab someone with one, or you may transfer some bad reagents into them!
+On a small ship, cooperation is key. Kind words or a friendly PDA e-mail can make a much more fulfilling round.
+Touching the Supermatter shard is good for health.
+Sometimes things don't go your way. Try and enjoy the ride, there's always next time - maybe your story helped make someone elses.
+Be polite. Be efficient. Have a plan to slip everyone you meet.
+Teaching your fellow spaceman is a time honored tradition. Help the new players out, pass the torch.
+Don't forget. You're here forever.
+A lot of things can be ground up to make chemicals and materials. Try and grind up a Danitos package, if you dare!
+Modular computers have a lot of utility. Buy some programs for it, or use the wall computers to send everyone in a department a message if they're not listening to the radio! Some can even be health scanners!
+As an antagonist, there are many ways to be engaging to the crew without being overtly violent - social engineering, blackmailing, stealing from the vault to give to the lower crew, sometimes a round can be more fun without murder.
+A character with Exploitables will show up with a red name in the Uplink. Fill out your Exploitable Information with a plausible story for your character - you are doing a public service for new antagonist players to work with.
+When there is no captain at the start, other heads of staff can become the acting-captain and secure important items such as the captain's spare and do job changes.
+Interacting with an alien artifact without proper protection can be hazardous for your health! Consider wearing anomaly protection gear before you end up in Medbay.
+On the Nerva, Cargo is in charge of Research and Development and Ship ammo manufaction, ensure a steady supply of torpedoes for the Nerva, and research for the Roboticist.
+As a Scientist, you have access to the Hadrian, which you can use to freely explore nearby planets, discover ancient ruins or dangerous ships, collect specimens, name caught aliens awful things, invent a reason as to why this alien artifact prints dead clowns, explore!
```

Comment:
As above, the map might not be the Nerva + downstreams. Perhaps we could add a system for map-specific tips in future?

Reply:
could make a map var for the tip config file. it'd give you duplicated tips across files but that's the most straightforward way of doing that. or add map specific tips only to map tip config files and pull from both the map specific one and the main tips.txt